orca-extensionpoint/orca-extensionpoint.gradle

```diff
@@ -0,0 +1,5 @@
+apply from: "$rootDir/gradle/spock.gradle"
+
+dependencies {
+  compile "com.google.code.findbugs:jsr305:3.0.2"
```

Comment:
Intentional? What is this dep buying us?

Reply:
this is the (AFAIK) canonical dep for `javax.annotation.Nonnull`